plugin-api/src/main/java/org/sonar/api/server/rule/RulesDefinition.java

```diff
@@ -256,8 +256,31 @@ public String label() {
     }
   }
 
+  enum MasvsVersion {
+    V1("MASVS", "masvs-1"),
+    V2("MASVS 2", "masvs-2"),;
+
+    private final String label;
+    private final String prefix;
+
+    MasvsVersion(String label, String prefix) {
+      this.label = label;
+      this.prefix = prefix;
+    }
+
+    public String label() {
+      return label;
+    }
+
+    public String prefix() {
+      return prefix;
+    }
+  }
+
   enum OwaspTop10Version {
-    Y2017("2017", "owaspTop10"), Y2021("2021", "owaspTop10-2021");
+    Y2017("2017", "owaspTop10"),
+    Y2021("2021", "owaspTop10-2021"),
+    Y2025("OWASP Top 10 2025", "owaspTop10-2025");
```

Comment:
```suggestion
    Y2025("2025", "owaspTop10-2025");
```
To match the existing convention.